knowledge-base/styling-tooltips-in-html5-based-report-viewers.md

```diff
@@ -106,12 +106,14 @@ You can provide styling for the tooltips in the **html**/**cshtml** page of your
 
 In case you need a custom solution for your tooltips, you can create your own ones, using the information provided in the **data-tooltip** attributes of the report viewer content. In the following example we will show you how to create tooltips using [jQueryUI Tooltip](https://jqueryui.com/tooltip/) widget:
     
- - First, declare the jQuery library in your **html** page:
+ - First, declare the __jQuery UI__ library in your **html** page. Here is a link to a CDN source:
 
 ```html
-<script src="http://code.jquery.com/ui/3.5.1/jquery-ui.min.js"></script>
+<script src="https://code.jquery.com/ui/1.13.0-rc.3/jquery-ui.min.js"></script>
```

Comment:
It's better to provide links to a stable version and generally it's recommended to use the earliest version that works. I guess in this case the 1.12.1 seems good.